### PR TITLE
Fixed ClassCastException

### PR DESCRIPTION
**Pull request by skabashnyuk**
Fixed ClassCastException when testing MethodInvokerFilter with everrest assured

### CODE REVIEW

File: everrest-assured/src/test/java/org/everrest/assured/MethodInvokerFilterTest.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    }
+
+    @Mock
+    private BookStorage bookStorage;
+
+    @InjectMocks
+    private BookService bookService;
+
+    @Test
+    public void shouldChangeMethodName() throws Exception {
```
Comment (sleshchenko) [inline everrest-assured/src/test/java/org/everrest/assured/MethodInvokerFilterTest.java]: fix test name

File: everrest-assured/src/main/java/org/everrest/assured/EverrestJetty.java [hunk 2]
```diff
@@ -233,7 +233,10 @@ private boolean isRestResource(Class<? extends Object> resourceClass) {
             pDescriptor.accept(rdv);
             return new TestResourceFactory<>(pDescriptor, testObject, field);
         } else if (clazz.getAnnotation(Filter.class) != null &&
-                   (RequestFilter.class.isAssignableFrom(clazz) || ResponseFilter.class.isAssignableFrom(clazz))) {
+                   (RequestFilter.class.isAssignableFrom(clazz)
```
Comment (sleshchenko) [inline everrest-assured/src/main/java/org/everrest/assured/EverrestJetty.java]: Let's remove these checks and then ApplicationPublisher will skip unsupported filters.
WDYT?

**Comment by garagatyi** [comment]
LGTM

File: everrest-assured/src/test/java/org/everrest/assured/MethodInvokerFilterTest.java [hunk 3]
```diff
@@ -0,0 +1,69 @@
+package org.everrest.assured;
```
Comment (sleshchenko) [inline everrest-assured/src/test/java/org/everrest/assured/MethodInvokerFilterTest.java]: license?

File: everrest-assured/src/test/java/org/everrest/assured/MethodInvokerFilterTest.java [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    }
+
+    @Mock
+    private BookStorage bookStorage;
+
+    @InjectMocks
+    private BookService bookService;
+
+    @Test
+    public void shouldBeAbleToGetBook() throws Exception {
```
Comment (sleshchenko) [inline everrest-assured/src/test/java/org/everrest/assured/MethodInvokerFilterTest.java]: shouldBeAbleToOverideArgumentByMethodInvokerFilter ?

**Pull request merged**